bsp/tm4c123bsp/board/board.c

```diff
@@ -0,0 +1,108 @@
+/*
+ * Copyright (c) 2006-2019, RT-Thread Development Team
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * Change Logs:
+ * Date           Author       Notes
+ * 2017-07-24     Tanek        the first version
+ * 2018-11-12     Ernest Chen  modify copyright
+ */
+ 
+#include <stdint.h>
+#include <stdbool.h>
+#include "hw_ints.h"
+#include "hw_sysctl.h"
+#include "hw_memmap.h"
+#include "hw_types.h"
+#include "fpu.h"
+#include "debug.h"
+#include "pin_map.h"
+#include "rom.h"
+#include "sysctl.h"
+#include "systick.h"
+#include "hw_ints.h"
+#include "board.h"
+
+// Holds the system core clock, which is the system clock 
+// frequency supplied to the SysTick timer and the processor 
+// core clock.
+uint32_t SystemCoreClock;
+
+
+
+void SystemCoreClockUpdate(void)
+{
+	
+	   FPULazyStackingEnable();
```

Comment:
please take care of code style, such as indentation.

Reply:
Thank you . I'v  format the code to obtain the code style.